.golangci.yml

```diff
@@ -39,24 +36,20 @@ linters:
     - gocyclo
     - gofmt
     - goimports
-    - golint
```

Comment:
Other projects use `revive` as a replacement for `golint`. Should we do the same in this project?